cmd/coap/main.go

```diff
@@ -9,6 +9,7 @@ import (
 	"log"
 	"os"
 
+	"github.com/mainflux/callhome/pkg/client"
```

Comment:
Probably we should add name here, like `chclient`, because `client` is too generic